Branches/Stable/Behaviors/obj_Miner.iss

```diff
@@ -1669,7 +1669,7 @@ objectdef obj_Miner
 					UI:UpdateConsole["Debug: Entity:WarpTo to ${LocationID} from Line _LINE_ ", LOG_DEBUG]
 					Entity[${LocationID}]:WarpTo[0]
 				}
-				if ${Universe[${LocationID}](exists)}
+				if ${Universe[${LocationID}](exists)} && ${Universe[${LocationID}].Name} != NULL

```

Comment:
NULL should not be possible in a LS String -- did you actually see this value?
Normal check would be (exists)